clib/src/main/resources/scala-native/errno.c

```diff
@@ -0,0 +1,11 @@
+#include <errno.h>
+
+int scalanative_errno() { return errno; }
```

Comment:
float.c and math.c below seem to use the scalanative_libc_ prefix.  That seems to be a useful convention.  If not relevant here, to keep changes to a minimum, perhaps in a spin-off PR?

Reply:
I think the `scalanative` prefix is good and we may need a `libc` too but this obviously is not totally agreed upon territory.